packages/eslint-config-eslint/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "allowSyntheticDefaultImports": false,
+        "esModuleInterop": false,
```

Comment:
Actually [**`esModuleInterop`**](https://www.typescriptlang.org/tsconfig/#esModuleInterop) is automatically set to `true` if [**`module`**](https://www.typescriptlang.org/tsconfig/#module) is set to **`Node16`** or **`NodeNext`**. But this makes me wonder if explicitly disable [**`esModuleInterop`**](https://www.typescriptlang.org/tsconfig/#esModuleInterop), would it still techinically be set to `true` since we have [**`module`**](https://www.typescriptlang.org/tsconfig/#module) set to **`NodeNext`**?

Reply:
Since it's just being used for type-checking and not transpilation, I'd say as long as there are no type errors it's 👍

Reply:
Awesome, so you still think I should remove them?

Reply:
I do, yup

Reply:
I tried it, removing them makes the type tests fail.

Reply:
In that case, I suppose I'm wrong to suggest removing it 😄